src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADDBOOK = 'ADDBOOK';
+const REMOVEBOOK = 'REMOVEBOOK';
+
```

Comment:
- [ ] Good job trying to adding your action with good names in uppercase. But as required for the project, kindly make sure it uses the [ducks pattern](https://github.com/erikras/ducks-modular-redux) to match the expected result and as each each action name is composed by 2 different words, I suggest you separate them with an underscore(`_`)  so to be something like this: `const REMOVE_BOOK = .../.../...`
* `

Reply:
Hello @tresorsawasawa 
Thank you so much for your review 🙏🙏